articles/active-directory/authentication/howto-registration-mfa-sspr-combined.md

```diff
@@ -46,6 +46,9 @@ If you have configured the Site to Zone Assignment List in Internet Explorer, th
 
 Securing when and how users register for Azure Multi-Factor Authentication and self-service password reset is now possible with user actions in Conditional Access policy. This feature is available to organizations who have enabled the [combined registration feature](../authentication/concept-registration-mfa-sspr-combined.md). This functionality may be enabled in organizations where they want users to register for Azure Multi-Factor Authentication and SSPR from a central location such as a trusted network location during HR onboarding.
 
+> [!NOTE]
+> This policy will only apply when a user navigates to a combined registration page.  This policy will not enforce MFA enrollment when accessing other applications.  You can create an MFA registration policy utilizing Azure Identity Protection [Azure Identity Protection - Configure MFA Policy](https://docs.microsoft.com/en-us/azure/active-directory/identity-protection/howto-identity-protection-configure-mfa-policy)
```

Comment:
I'd suggest the following to remove the _en-us_ locale from the link and future-proof things for other environments. Otherwise looks good

```suggestion
> This policy only applies when a user navigates to a combined registration page.  This policy doesn't enforce MFA enrollment when accessing other applications.  You can create an MFA registration policy utilizing Azure Identity Protection [Azure Identity Protection - Configure MFA Policy](../identity-protection/howto-identity-protection-configure-mfa-policy.md)
```